mobius-rx/src/test/java/com/spotify/mobius/rx/RxMobiusLoopTest.java

```diff
@@ -85,11 +87,30 @@ public void shouldPropagateIncomingErrorsAsUnrecoverable() throws Exception {
   @Test
   public void startModelAndEffects() {
     RxMobiusLoop<Integer, String, Boolean> loop =
-        new RxMobiusLoop<>(factory, "StartModel", ImmutableSet.of(true, false));
+        new RxMobiusLoop<>(builder, "StartModel", ImmutableSet.of(true, false));
     final AssertableSubscriber<String> subscriber = Observable.just(1).compose(loop).test();
     subscriber.assertValue("StartModel");
     subscriber.assertNoErrors();
     assertEquals(2, connection.valueCount());
     connection.assertValues(true, false);
   }
+
+  @Test
+  public void shouldSupportStartingALoopWithAnInit() throws Exception {
```

Comment:
Would it make sense to cover the error case here too, or do you think it'd be overkill since it's already covered in the loop tests?

Reply:
Good point, done.